README.md

```diff
@@ -192,6 +192,9 @@ cargo run --features python --bin metta-repl
 ```
 You can also find executable at `./target/debug/metta-repl`.
 
+Please be aware that running REPL with Python support is not compatible with
```

Comment:
I agree to put it in other words, on the other hand from my experience it is vice versa. People who use Python virtual environments are used they works without issues. Mostly because they usually use software packed for this environments.